addus

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+if [ -d "$HOME/.ssh" ]
+then
+    echo 'ssh dir exists, proceeding...'
+else
+    echo 'ssh dir does not exist, please generate it with "ssh-keygen -t ecdsa" for example'
+    ssh-keygen -b 521 -N "" -t ecdsa -f $HOME/.ssh/id_ecdsa
+fi
+echo 'This is for Webhosting.coop dev only, this will add our keys to your server, BEWARE!'
+touch $HOME/.ssh/authorized_keys
+TMP=$(mktemp -d --suffix=SDX)
+cp $HOME/.ssh/authorized_keys $TMP/mysshauthorized_keys
+curl --silent https://raw.githubusercontent.com/WebHostingCoopTeam/keys/master/keys |xargs -n1 -I{} echo '{}'>> $TMP/mysshauthorized_keys
```

Comment:
Should we have this pointing at out own directory or is this here as we don't currently have such a directory?

Reply:
This is a valid mishap, I thought I had replaced that line.   I may have reverted that file errantly.   Correcting shortly.